views/index.erb

```diff
@@ -17,25 +53,125 @@
 
   <div id="problems" class="box">
     <h3>Programming Problems</h3>
-    <a href="/primes">My Prime Number Solution</a>
+    <a href="/primes">My Prime Number Solution</a><br />
+    <a href="/palindrome">Largest Palindrome Product</a><br />
+    <a href="/pandigital">Pandigital Products</a><br />
   </div>
 
-  <div class="box">
-    <h3>Edit Profile</h3>
 
+  <div id="editbox" class="box">
+   <hr />
+   <br /><br />
+    <h3>Edit Profile</h3>
     <form action="/update" method="POST">
       <fieldset>
         <div class="form-group">
           <label for="trainee[about]">About</label>
-          <input type="text" name="trainee[about]" value="<%= @profile["about"] %>"/>
+          <textarea name="trainee[about]" rows="10" cols="100"><%= @profile["about"] %></textarea>
+          <br />
+          <div class="wordcount">Words: <span id="about_wordcount"></span></div>
         </div>
-
+        <br />
         <!-- TODO - Add more fields so that your profile can be edited -->
 
+        <div class="form-group">
+          <label for="trainee[ambition]">Ambition</label>
+          <textarea name="trainee[ambition]" rows="10" cols="100"><%= @profile["ambition"] %></textarea>
+          <div class="wordcount">Words: <span id="ambition_wordcount"></span></div>
+        </div>
+        <br />
```

Comment:
You're relying on `br` tags a lot for spacing. Investigate how you could better add spacing using CSS margins or padding on your `.form-group` elements and elsewhere.